### ISSUE TITLE
Skipped tests displayed passed in JUnit xml

### ISSUE THREAD

**Issue by Montimort** [xunit/xunit#2979]
When having Fact(Skip = "something")
in "xUnit" xml, this is correctly flagged as skipped, in jUnit style xml, this is marked as passed

A hint how this could be correctly implemented: https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd

**Comment by bradwilson** [comment]
Thanks. The original provided JUnit XSL-T was provided about 6 years ago, and by the looks of it the author based it the output from NUnit. I used https://github.com/testmoapp/junitxml?tab=readme-ov-file as my guide, since it was more readable than an XSD file (and seemed to be updated slightly more recently).

**Comment by bradwilson** [comment]
- Available in v2 `2.9.1-pre.8`
- Available in v3 `0.2.0-pre.40`

https://xunit.net/docs/using-ci-builds

**Comment by bradwilson** [comment]
Here's an example of updated output, which includes a trait, plus a warning from a v3 run:

```xml
<?xml version="1.0" encoding="utf-8"?>
<testsuites name="Test results" time="0.158" tests="3" failures="1" errors="0" skipped="1">
  <testsuite name="Test collection for Empty.TestClass (id: bb50796f4a69d677b2503126e7ed72cbf1c958294d7d7f1ba49b543e57bfa155)" time="0.0
22" tests="3" failures="1" skipped="1">
    <testcase name="Empty.TestClass.Failing" classname="Empty.TestClass" time="0.01985">
      <failure type="Xunit.Sdk.FailException" message="This is a failing test">   at Xunit.Assert.Fail(String message) in D:\a\xunit\xun
it\src\xunit.v3.assert\Asserts\FailAsserts.cs:line 34
   at Empty.TestClass.Failing() in C:\Dev\repro\empty\UnitTest.cs:line 17</failure>
    </testcase>
    <testcase name="Empty.TestClass.Passing" classname="Empty.TestClass" time="0.0024541">
      <system-out>Test output
</system-out>
      <properties>
        <property name="trait:category" value="interesting" />
        <property name="warning" value="This is a warning" />
      </properties>
    </testcase>
    <testcase name="Empty.TestClass.Skipping" classname="Empty.TestClass" time="0">
      <skipped message="This is a skipped test" />
    </testcase>
  </testsuite>
</testsuites>
```
